tests/e2e/config/wdio.android.rdc.conf.js

```diff
@@ -10,7 +10,7 @@ config.capabilities = [
     deviceName: 'Google Pixel',
     automationName: 'UiAutomator2',
     // The reference to the app
-    testobject_app_id: '2',
+    testobject_app_id: '3',
```

Comment:
Curious why this was incremented - what is this ID, how does it change, and how do I determine it?

Reply:
It's the last loaded app id in the Test object cloud, I can give you a demo about it